modules/administration-guide/partials/proc_configuring_bitbucket_authentication.adoc

```diff
@@ -54,6 +54,10 @@ The main parts of the secret are:
 
 |===
 
+NOTE: Please note that encoding string into base64 format using *base64* tool on Linux machines most probably will
+add the newline character to the end of source string, so hash may become unusable as the authentication header value after decoding.
+To avoid that, it is better to use online tools, or strip newlines explicilty (for example, using *tr -d \\n*)
```

Comment:
you can use `base64 -w0` and output will be without new lines

Reply:
@sparkoo Hi Michal, can I add the `base64 -w0` as a secondary option to `tr -d \\n`? Thx